### PR TITLE
Add RemoteError to multiprocessing.managers

### PR DESCRIPTION
**Pull request by euresti**

### CODE REVIEW

File: stdlib/3/multiprocessing/__init__.pyi [hunk 1]
```diff
@@ -2,6 +2,8 @@
 
 from typing import Any, Callable, Iterable, Mapping
 
+from multiprocessing.process import current_process as current_process
```
Comment (matthiaskramm) [inline stdlib/3/multiprocessing/__init__.pyi]: mypy can't find `multiprocessing.process`.

Comment (euresti) [inline stdlib/3/multiprocessing/__init__.pyi]: Hmm. It's there

```
$ cat stdlib/3/multiprocessing/process.pyi 
def current_process(): ...
def active_children(): ...
```

Am I mispelling it?  Or do I have to import it in a different way?

Comment (euresti) [inline stdlib/3/multiprocessing/__init__.pyi]: Oh wait. It's there locally.  When did I add it.

**Comment by euresti** [comment]
Now with the missing file. Sorry!

**Pull request merged**